.github/workflows/docker_build_base.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        branch: [ latest, mysql57, percona57, percona80 ]
+        branch: [ latest, percona80 ]
```

Comment:
Why does this not have mysql80?

Reply:
For the base image we do not publish a `vitess/base:v19.0.6-mysql80` image. We only publish the following:

```
vitess/base:v19.0.6
vitess/base:v19.0.6-percona80
```

Starting in v20 we got rid of the base image, I would not suggest fixing this irregularity since v19 will be dropped in 5 months.